react/src/js/components/Consonant/Search/SearchIcon.jsx

```diff
@@ -25,6 +25,7 @@ const SearchIcon = (props) => {
     return (
         <button
             data-testid="search-icon"
+            aria-label="Search"
```

Comment:
Need to be localiized.

Reply:
We are creating a separate ticket to localize all aria-labels

Reply:
done!